### ISSUE TITLE
include cards into the "Search for .. in all boards" if search would find a comment

### ISSUE THREAD

**Issue by kinimodmeyer** [nextcloud/deck#2979]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the solution you'd like**

could we show the cards under "Search for ... in all boards" if the search would find a comment of this card?
at the moment it is a little bit strange that the "new" input field search give you less results then the unified search of the deck app (& fills the input on search).

**Comment by juliushaertl** [comment]
Yes, that would be indeed a nice enhancement. @jancborchardt or @Clementine46 maybe you have an idea how we could nicely integrate comment search results into the global card search:

![image](https://user-images.githubusercontent.com/3404133/114740365-11ff0b00-9d4a-11eb-926b-156fdca568ef.png)

**Comment by rostmodern** [comment]
this is my main culprit with this wonderful app!
any news?
